<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="commandsTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>571</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addCharButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add character</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCondButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>240</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add condition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delCharButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete character</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delCondButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete condition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inpButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Input string</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="lineTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>741</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>450</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>■</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x1Button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>x1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x2Button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>x2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x3Button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>510</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>x3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x4Button">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>510</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>x4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x5Button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>510</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>x5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="incButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>510</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>420</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="maxButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>570</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>UNLIMITED POWER!!!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
